VVDocumenter-Xcode/Setting/VVDSettingPanelWindowController.m

```diff
@@ -54,6 +56,11 @@ - (void)windowDidLoad
     self.btnUseSpaces.state = (NSCellStateValue)[[VVDocumenterSetting defaultSetting] useSpaces];
 
     self.btnAddSinceToComment.state = (NSCellStateValue)[[VVDocumenterSetting defaultSetting] addSinceToComments];
+    self.mtxSinceOptions.enabled = [[VVDocumenterSetting defaultSetting] addSinceToComments];
+    [self.mtxSinceOptions selectCellAtRow:(NSInteger)[[VVDocumenterSetting defaultSetting] sinceOption] column:0];
+    self.tfSinceVersion.enabled = [[VVDocumenterSetting defaultSetting] addSinceToComments];
+    self.tfSinceVersion.stringValue = [[VVDocumenterSetting defaultSetting] sinceVersion];
```

Comment:
I changed the `-sinceVersion:` to return an empty string if no version has been stored on initial `-windowDidLoad:`

https://github.com/onevcat/VVDocumenter-Xcode/pull/169/files#diff-d485c3d9b020ac2840b8995bc1b8f991R176